invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -122,6 +121,17 @@ const mapFormFields = (obj, parentField, isCreate, formFieldsConfig, dropDumpOnl
       );
     }
 
+    if (fieldSchema.type === "datetime") {
```

Comment:
I think we don't need this conditional statement since in line 36 we added the description. I would prefer to avoid adding more if blocks here